src/FSharpVSPowerTools.Core/Utils.fs

```diff
@@ -1,4 +1,42 @@
-﻿namespace FSharpVSPowerTools
+﻿namespace FSharpPowerTools.Core
+
+type FileName = string
+
+[<Measure>] type FCS
+
+type Point<[<Measure>]'t> = { Line : int; Column : int }
+type Range<[<Measure>]'t> = { From : Point<'t>; To: Point<'t> }
```

Comment:
I cannot find any usages of these types specified with other than `FCS` measure. What's the point in measures here?